### PR TITLE
Suggest  springcloud openjdk parent image for building task apps

### PR DESCRIPTION
**Pull request by tzolov**

### CODE REVIEW

**Comment by trisberg** [comment]
Some of the suggested dependencies to add need version, I got this hist coping from the docs:

```
'dependencies.dependency.version' for io.micrometer.prometheus:prometheus-rsocket-spring:jar is missing.
```

File: content/documentation/pages/5-feature-guides/3-batch/4-monitoring.md [hunk 1]
```diff
@@ -52,6 +52,17 @@ Similarly to enable Prometheus metrics collection using RSocket the following de
 </dependency>
 ```
 
+<!--TIP-->
+
+For building a Task Docker image it is recommended to extend from the `springcloud/openjdk:latest` image. E.g. your task `Dockerfile` should start like this:
```
Comment (sabbyanandan) [inline content/documentation/pages/5-feature-guides/3-batch/4-monitoring.md:57]:
```suggestion
To build a Docker image, you could extend from the `springcloud/openjdk:latest` base-image. For example, your task `Dockerfile` could start like this:
```

**Comment by sabbyanandan** [comment]
@trisberg: Could you review/merge this, please?

**Comment by sabbyanandan** [comment]
@tzolov: let's wrap this one when you get a chance.

**Comment by tzolov** [comment]
Rebased an merged at: 604358dbee220811fc5ecaf1fc41e1d133042441

**Pull request closed without merging**